easy-ui-react/src/Banner/Banner.module.scss

```diff
@@ -0,0 +1,77 @@
+@use "../styles/common" as *;
+
+.variantSuccess {
+  @include component-token(
+    "banner",
+    "color",
+    theme-token("color.text.success.inverse")
+  );
+  @include component-token(
+    "banner",
+    "background.color",
+    theme-token("color.background.success")
+  );
+}
+
+.variantPrimary,
+.variantNeutral {
+  @include component-token(
+    "banner",
+    "color",
+    theme-token("color.text.inverse")
+  );
+}
+
+.variantPrimary {
+  @include component-token(
+    "banner",
+    "background.color",
+    theme-token("color.background.primary")
+  );
+}
+
+.variantNeutral {
+  @include component-token(
+    "banner",
+    "background.color",
+    theme-token("color.background.neutral")
+  );
+}
+
+.Banner {
+  @include component-token(
+    "banner",
+    "border_radius",
+    design-token("shape.border_radius.base")
+  );
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0;
+  flex-wrap: wrap;
+  white-space: pre-wrap;
+  width: 100%;
+  padding: design-token("space.1") design-token("space.2");
+  color: component-token("banner", "color");
+  background-color: component-token("banner", "background.color");
+
+  @include breakpoint-sm-up {
+    flex-direction: row;
+    padding: design-token("space.1") design-token("space.3");
+    border-radius: component-token("banner", "border_radius");
+  }
+
+  @include breakpoint-md-up {
+    padding: design-token("space.1") design-token("space.4");
+  }
+}
+
+.bannerContentContainer {
+  display: flex;
+  max-width: 1348px;
```

Comment:
**question:** was there a reason for creating a separate container for this instead of setting it on the `Banner` itself? i believe it should work fine on the root element instead. (it may be acting up with the left/right padding but once the `box-sizing` is set correctly to `border-box`, which i updated in a recent diff, it should be good)